Stylesheets/Coding Style.md

```diff
@@ -0,0 +1,97 @@
+# Coding Style
+
+We mainly use SASS (mainly written as SCSS) as a pre-processor to generate the final CSS styles for each project. As a result, our CSS coding style mixes with the SASS and on the following document we try to give this distinction by pointing to what only applies to SASS (in general because it is invalid CSS).
+
+## General principles
+
+If you follow nothing else at least don't forget these next few lines.
+
+> "Part of being a good steward to a successful project is realizing that writing code for yourself is a Bad Idea™. If thousands of people are using your code, then write your code for maximum clarity, not your personal preference of how to get clever within the spec." - Idan Gazit
+
+* **All code should look like a single person wrote it, even when many actually did.** This ensures that anyone coming in to the project later on for whatever reason (correct a bug, extend functionality, help with development, etc.) will be able to understand and contribute in a single voice.
+* Code readability should come first and optimization second.
+* When in doubt about a style raise the question for debate and proceed with existing common patterns (share these patterns when with your question).
+* Enforce the agreed-upon style, always. Code review is a particular example where this is almost mandatory.
+
+## Syntax an Formatting
+This probably the main reason why code feels like it was written by different people. By keeping the syntax and formatting the code in a clean way maintaining and debugging the code become easier tasks.
+
+### Syntax
+* Write one selector per line
```

Comment:
I actually like bundling a few instructions on each example, but this part is not yet set in stone. I realised it would take too long to consider each point with an example, so I'm trying to have the information structured and then I will revise the examples.
Also, turning everything into headings seems to me to break the semantics, because very little "normal" text exists in most cases.